storage/libradb/src/ledger_store/mod.rs

```diff
@@ -42,7 +42,7 @@ pub(crate) struct LedgerStore {
 }
 
 // TODO: Either implement an iteration API to allow a very old client to loop through a long history
-// or guarantee that there is always a recent enough waypoint and client knowns to boot from there.
+// or guarantee that there is always a recent enough waypoint and client knows to boot from there.
```

Comment:
@zekun000 I'm sure we have a plan to deal with this situation, right?

Reply:
I don't have a plan for client yet, but for validators it'll keep requesting epoch change proof if it receives future epoch messages until it catches up

Reply:
hmm, I realize the behavior is not returning at most 100 items but return error if it exceeds 100, and there's no API for client to pass the limit

seems there's no way to loop through from very far in this case?

Reply:
Can we set a limit with a tag like EOF telling user whether we reach the latest?

Reply:
@zekun000 Yes, so the plan is the "looping" instead of "guaranteeing there is a recent enough waypoint"? We need to implement the API with cursor.

@lightmark a user can compare the epoch in the last epoch change li with the one on the latest li to know if it has reached the latest.